.pipelines/nuget-publishing.yml

```diff
@@ -27,12 +27,17 @@ parameters:
 - name: ort_version
   displayName: 'OnnxRuntime version'
   type: string
-  default: '1.17.3'
+  default: '1.18.0-dev-20240426-1256-b842effa29'
+
+- name: ort_cuda_version
```

Comment:
Later on we will merge the three versions into a single one.

Reply:
Yeah, once we figure out how to do it in ORT